providers/ses/src/lib/ses.provider.ts

```diff
@@ -25,6 +25,28 @@ export class SESEmailProvider implements IEmailProvider {
     });
   }
 
+  private async sendMail({ html, text, to, from, subject, attachments }) {
+    const transporter = nodemailer.createTransport({
+      SES: { ses: this.ses, aws: { SendRawEmailCommand } },
+    });
+    let info;
+
+    try {
+      info = await transporter.sendMail({
+        html,
+        text,
+        to,
+        from,
+        subject,
+        attachments,
+      });
+    } catch (err) {
+      info = err;
+    }
```

Comment:
Want this mask the error when thrown from the `sendMessage` command so the usecase of the trigger will actually report a success wrongly. I would suggest to remove this from here (the catch), and only add it inside the testIntegration method. 

@L-390 @BiswaViraj

Reply:
So more like this?
`
try{
testResponse = await this.sendMail({
      html: '',
      text: 'This is a Test mail to test your Amazon SES integration',
      to: this.config.from,
      from: this.config.from,
      subject: 'Test SES integration',
      attachments: {},
    });
}
catch(err)
{
testResponse = err;
}
}`

Reply:
Yes, meaning that the catch will only happen in the checkIntegration method :)